demo-certs/create_certs.sh

```diff
@@ -22,6 +22,8 @@ HONO_TRUST_STORE=trustStore.jks
 HONO_TRUST_STORE_PWD=honotrust
 AUTH_SERVER_KEY_STORE=authServerKeyStore.p12
 AUTH_SERVER_KEY_STORE_PWD=authkeys
+DEVREG_SERVER_KEY_STORE=devRegServerKeyStore.p12
```

Comment:
Yes, you are probably right - just did not want to use too long names.
But I agree, will provide an update with the long name :+1: